src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerTestingControllerFunctionalTest.java

```diff
@@ -72,7 +73,7 @@ public void given_post_event_using_test_rest_endpoints_should_create_task() {
             .body("MessageId", equalTo(messageId))
             .body("Sequence", notNullValue())
             .body("CaseId", equalTo(caseIdForTask))
-            .body("EventTimestamp", equalTo(timeStampString))
+            .body("EventTimestamp", equalTo(timeStamp.toString()))
```

Comment:
Since this is an internal rest API it doesn't matter as much, however the format of timestamps is important.  We should keep it consistent with the other endpoints.